src/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -365,25 +365,167 @@ public class Options
 
         public sealed class Connection : IDisposable
         {
+            private const int _blockSize=4000;
```

Comment:
Nit: spacing

Reply:
Also, consts are usually capitalized without _, like so: BlockSize. And maybe call this BufferSize?